skops/card/tests/test_card.py

```diff
@@ -405,6 +428,69 @@ def test_with_metadata(self, card: Card, meth):
         assert result == expected
 
 
+class TestModelCardFromPath:
+    @pytest.mark.parametrize("suffix", [".pkl", ".pickle", ".skops"])
+    def test_model_card_str(self, suffix):
+        model0 = LinearRegression(n_jobs=123)
+        save_file_str = save_model_to_file(model0, suffix)
+        card_from_str = Card(save_file_str)
+        card_from_model0 = Card(model0)
+
+        assert card_from_model0.model.n_jobs == card_from_str.model.n_jobs
+
+    @pytest.mark.parametrize("suffix", [".pkl", ".pickle", ".skops"])
+    def test_model_card_path(self, suffix):
+        model0 = LinearRegression(n_jobs=123)
+        save_file = save_model_to_file(model0, suffix)
+        save_file_path = Path(save_file)
+        card_from_path = Card(save_file_path)
+        card_from_model0 = Card(model0)
+
+        assert card_from_model0.model.n_jobs == card_from_path.model.n_jobs
+
+
+class TestCardModelAttribute:
```

Comment:
I would like to see an additional test case here, namely that the model is only loaded once.

One way to test this would be to save the model to a file, pass the path to `Card`, then accessing `card.model` should return the model. After this, delete the file and access `card.model` again. This way, if the implementation accidentally loads on each access, the test would raise an error.

Maybe there is a better way to test this, but this seems more robust than, say, patching `_load_model`.